pyproject.toml

```diff
@@ -26,7 +26,7 @@ select = ["E", "W", "F", "C", "N", "R", "D", "H"]
 # Ignore D107 Missing docstring in __init__
 ignore = ["W503", "E501", "D107"]
 # D100, D101, D102, D103: Ignore missing docstrings in tests
-per-file-ignores = ["tests/*:D100,D101,D102,D103,D104,D205,D212,D415,DCO010", "src/chrony.py:DCO060"]
+per-file-ignores = ["tests/*:D100,D101,D102,D103,D104,D205,D212,D415,DCO010,DCO020,DCO030", "src/chrony.py:DCO060"]
```

Comment:
I think these are the same rules for ignoring missing docstrings in tests, but somehow `tests/unit/utils.py` is not causing any linting errors, probably because the filename doesn't have 'test' in it. I'll adjust the comment here, thanks!

Reply:
So `D100,D101,D102,D103,D104,D205,D212,D415,DCO010,DCO020,DCO030` is all about missing docstrings? and `DC060` for chrony.py?